docs/reference/modules/admin-bar.md

```diff
@@ -80,11 +80,11 @@ export default () => {
       return;
     }
     // Custom code for button action
-  };
+  });
 }
 ```
   <template v-slot:caption>
-    /modules/myCustomModule/ui/apos/index.js
+    /modules/myCustomModule/ui/apos/apps/index.js
```

Comment:
Every `.js` file in `ui/apos/apps` is an entry point in itself, so the convention is to name them by what they do, not `index.js`. Examples include `AposBusy.js`, `AposAreas.js`, etc. A third party should not use the `Apos` prefix though. I can't really suggest a name here because there isn't enough of a suggested use case being gestured at in this case, but... uppercase first letter of the name, and not `Index`

Reply:
Changed.